### ISSUE TITLE
Conflicting requirements for types-aiobotocore 2.15.2.post1

### ISSUE THREAD

**Issue by lg-kialo** [youtype/mypy_boto3_builder#327]
The latest release of `types-aiobotocore` which is `2.15.2.post1` has the following requirements:
```
"aiobotocore": ["aiobotocore==2.15.2", "botocore==1.35.54"]
```

but aiobotocore 2.15.2 [requires](https://github.com/aio-libs/aiobotocore/blob/2.15.2/pyproject.toml#L33) `"botocore >=1.35.16, <1.35.37"` so I don't see how it can be used.

For reference, `types-aiobotocore 2.15.2` had the following requirements instead
```
"aiobotocore": ["aiobotocore==2.15.2", "botocore==1.35.36"]
```

**Comment by vemel** [comment]
Thank you for the report. Let me re-release the package.

**Comment by vemel** [comment]
Published `types-aiobotocore-lite 2.15.2.post2` and `types-aiobotocore 2.15.2.post2`. Let me know if the issues is solved.

**Comment by lg-kialo** [comment]
Thank you!
